doc/source/kubernetes/amazon/step-zero-aws.md

```diff
@@ -31,9 +31,18 @@ template you will use to setup and shape your cluster.
 
    When creating it, assign the IAM role created in step 1.
 
-   Once created, download ssh keys.
+   Once created, download ssh keys (.pem file). Ensure permissions are restrictive on the file:
+
+   ```
+   chmod 400 name.pem
+   ```
+
+3. SSH to your CI host as:
+
+   ```
+   ssh -i /path/name.pem ec2-user@my-instance-Private-IPv4-address
```

Comment:
This may be specific to my required way of ssh’ing in

Reply:
It must be! A private IP shouldn't be accessible from outside AWS.

Reply:
Is there an request for change here @manics?

If ec2-user isn't a hardcoded username, perhaps you can make it included `my-` as well to clarify its really not something predefined by AWS that others can re-use.

Reply:
`my-instance-Private-IPv4-address` won't work, you shouldn't be able to SSH into your private IP from outside! Unless you've got a VPN or something more advanced.

`ec2-user` is fine if you're using the Amazon Linux AMI since that's the default username.... So this depends on how much detail you want to go into. If you assume people know how to use AWS EC2 it's enough to just say bring up a VM and SSH in, if you don't then you have to go into details about the choice of AMI, security groups to allow SSH, etc.

Reply:
Thanks for the comments. I'll scale my code back a bit and point to documentation. It saves on being too specific